ISSUE_LIFECYCLE.md

```diff
@@ -0,0 +1,37 @@
+# Issue Lifecycle
+
+In an effort to ensure a balance between work carried out internally by the NGINX engineering team, while simultaneously encouraging community involvement on this project, we use the following issue lifecycle. (Note: The issue *creator* refers to the community member that created the issue. The issue *owner* refers to the NGINX team member that is responsible for managing the issue lifecycle.)
```

Comment:
```suggestion
To ensure a balance between work carried out by the NGINX engineering team while encouraging community involvement on this project, we use the following issue lifecycle. (Note: The issue *creator* refers to the community member that created the issue. The issue *owner* refers to the NGINX team member that is responsible for managing the issue lifecycle.)
```
Omitting needless words. Work by the engineering team is implicitly internal by nature; no need to use the word simultaneously, since it doesn't add anything to the sentence - it fully contains the context about why the lifecycle exists at all.